contracts/MerkleLib.sol

```diff
@@ -51,6 +51,18 @@ library MerkleLib {
         uint256[] refundAmounts;
     }
 
+    // TODO: should these structs be in this file?
+    struct RelayData {
```

Comment:
would a better name be `SlowRelayFulfilment`? then use this through out, such as `verifySlowRelayFufilment`

Reply:
Well, it's tricky here because this is _the exact_ data structure the SpokePool uses internally to represent a relay. I don't know if we want to rename that struct.

Reply:
move this to SpokePool interface?

Reply:
hhhmm. ye I see your point. not ideal.

if we move it to the `SpokePool` interface then we'll get the MerkleLib to import from this interface, yes? equally, we can pull the Hub's Merkle `structs` to a `HubPool` interface and import them into the `MerkleLib` in the same way.

Reply:
Renamed!